src/library/playlisttablemodel.cpp

```diff
@@ -275,6 +279,11 @@ void PlaylistTableModel::moveTrack(const QModelIndex& sourceIndex,
     }
 
     m_pTrackCollectionManager->internalCollection()->getPlaylistDAO().moveTrack(m_iPlaylistId, oldPosition, newPosition);
+
+    if (oldPosition==1 || newPosition==1) {
+        qDebug() << "First track changed";
```

Comment:
IMO we don't need this debug message here, the one in AutoDJProcessor is sufficient.

Reply:
clang-format is complaining here.
You can avoid these complains by installing 
pre-commit 
https://github.com/mixxxdj/mixxx/wiki/Using-Git#set-up-automatic-code-checking